MKOB/kobmain.py

```diff
@@ -136,12 +138,26 @@ def update_sender(id):
 
 def readerCallback(char, spacing):
     """display characters returned from the decoder"""
-    n = int(spacing + 0.5)
-    if n > 5:
-        sp = "__" if char != "~" else ""
+    if kc.CodeType == 0:  # TODO: use config enum for American
+        sp = (spacing - 0.25) / 1.25  # adjust for American Morse spacing
+    else:
+        sp = spacing
+    if sp > 5:
+        txt = "__" if char != "~" else ""
+    elif sp < -0.2:
+        txt = ""
+    elif sp < 0.2:
+        txt = NNBSP
+    elif sp < 0.5:
+        txt = 2 * NNBSP
+    elif sp < 0.8:
+        txt = NNBSP + " "
     else:
-        sp = n * " "
-    ka.codereader_append(sp + char)
+        n = int(sp - 0.8) + 2
+        txt = n * " "
+    txt += char
+    ka.codereader_append(txt)
+##    print("kobmain: {:.2f} '{}'".format(spacing, txt))  # TODO: temporary
```

Comment:
Leftover debug code?  Might as well get rid of it now?

Reply:
You have a sharp eye! Yes, that was debugging code. I almost deleted it before doing the commit, but I suspect I'll be revisiting the proportional spacing issue again and will likely want to reenable that statement. I'll be sure to clean it up when I'm done with it.

My plan is to use this version of MKOB on wire 11 during this Saturday's net and see how everything works. Don't be surprised if you see a slew of new issues get posted as a result!

All in all, I'm very happy with the way things are coming along. Having said that, I know there's a bunch of fine tuning that we'll want to do.

Reply:
When/what is "Saturday's net"?

I would like to join if possible.

Reply:
Maybe (hopefully) I'll have an installer by then. It's kind of tough work (creating and then testing on Win/Mac/LX).